### PR TITLE
Not found fix

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by thedillonb** [comment]
Awesome fix. I know it's merge conflicted but I'll resolve it and merge it in manually.

Thanks!

File: worker/index.js [hunk 1]
```diff
@@ -26,28 +26,28 @@ function processRecord(record, callback) {
 
     misc.processRegistration(client, updatedDate, function(err, lastModified, results) {
         if (err) {
-            console.error(err);
+            console.error('Error procesing registrations: %s - %s', err, err.stack);
```
Comment (phetmanyptn) [inline worker/index.js]: Deramleague

**Comment by phetmanyptn** [comment]
Hack Deramleague sccoger

**Comment by reinier19** [comment]
V

**Comment by jslee1301** [comment]
So, now its found? Hmm

File: worker/index.js (hunk 1 above)
Comment (doanbahai) [inline worker/index.js]: doanbahai12591@gmail.com

**Comment by kincaider** [comment]
Los

**Comment by kincaider** [comment]
Conflicting files
lib/github.js
worker/index.js

File: worker/index.js [hunk 2]
```diff
@@ -26,28 +26,28 @@ function processRecord(record, callback) {
...
+            });
         }
 
-        _.each(results, function(result) {
-            //console.log('pushing to %s: %s', record.tokens, result.msg);
-            push.send(record.tokens.split(','), result.msg, result.data);
-        });
-
         db.updateUpdatedAt(record.oauth, record.domain, lastModified, function(err) {
             if (err) console.error(err);
```
[the remark this answers is not in the document]
Comment (zorderg) [reply worker/index.js]: @https://vk.com/annatagunova

File: worker/index.js (hunk 1 above)
Comment (zorderg) [reply worker/index.js]: Yhu

**Comment by loctruong0104** [comment]
Like

**Pull request closed without merging**